core/src/main/java/io/seata/core/model/BranchStatus.java

```diff
@@ -108,7 +108,7 @@ public int getCode() {
         return code;
     }
 
-    private static final Map<Integer, BranchStatus> MAP = new HashMap<>(values().length);
+    private static final Map<Integer, BranchStatus> MAP = new HashMap<>((int) ((float) values().length / 0.75F + 1.0F));
```

Comment:
The capacity of the map is always 2 ^ n.

Reply:
tks，min value is 2^n。So 10 won't expand, if it's 12 it will.

Reply:
`(int) ((float) values().length / 0.75F + 1.0F)  `
It should be written in a more correct way

Reply:
I don't recommend that. 0.75 is a hashMap default factor, should not directly write this value here,  if 0.75 becomes a different value in a later JDK version, what shall we do?

Reply:
It's supposed to be a constant, in theory it doesn't change。JDK Hashmap Source Code is optimized all the time, but this load factor has not changed

Reply:
We encapsulate a method to handle internal logic, and default load factors can also be defined as variables. Forget it. I close it.